README.md

```diff
@@ -20,9 +20,10 @@ var b = 3;
 a = b;
 ```
 
+
 After this code executes, what are the values of a and b? Please explain your answer.
 
-<!-- Replace this comment with your answer -->
+a and b are 3.  the variable on the right evaluates to the variable on the left.  if b = 2 and a = 3, then both would be 2.
```

Comment:
Your answer is correct, but this explanation doesn't make sense. Can you clarify it at all?